ui/models/job.js

```diff
@@ -101,11 +101,9 @@ export default class JobModel {
     const isManyJobs = jobIds.length > 1;
```

Comment:
`isManyJobs` is only ever used to determine the phrase to include in the message.
How about removing `isManyJobs` and using this instead to avoid duplication?

```js
const jobTerm = jobIds.length > 1 ? 'jobs' : 'job';
```
(there may be a better name than `jobTerm`)